.ruby-version

```diff
@@ -1 +1 @@
-3.2.0
```

Comment:
This looks like it slipped in. Probably should be a separate PR (especially as it's a downgrade)!

Reply:
Good catch. It shouldn't be needed at all. It'll sit here anyway until the k9s move is done.